main/admin/session_add.php

```diff
@@ -491,6 +156,298 @@ function emptyDuration() {
+
+    $sql="SELECT user_id, lastname, firstname, username FROM $tbl_user "
+        . "WHERE status = '1' "
+        . $orderClause;
+
+    if (api_is_multiple_url_enabled()) {
+        $userRelAccessUrlTable = Database::get_main_table(TABLE_MAIN_ACCESS_URL_REL_USER);
+        $accessUrlId = api_get_current_access_url_id();
+        
+        if ($accessUrlId != -1) {
+            $sql = "SELECT user.user_id, username, lastname, firstname FROM $tbl_user user "
+                . "INNER JOIN $userRelAccessUrlTable url_user ON (url_user.user_id = user.user_id) "
+                . "WHERE access_url_id = $accessUrlId AND status = 1 "
+                . $orderClause;
+        }
+    }
+
+    $result = Database::query($sql);
+	$coachesList = Database::store_result($result);
+
+    $coachesOptions = array();
+
+    foreach($coachesList as $coachItem){
+        $coachesOptions[$coachItem['username']] = api_get_person_name(
+            $coachItem['firstname'],
+            $coachItem['lastname']
+        ).' ('.$coachItem['username'].')';
+    }
+
+    $form->addElement('select', 'coach_username', get_lang('CoachName'), $coachesOptions, array(
+        'id' => 'coach_username',
+        'class' => 'chzn-select',
+        'style' => 'width:350px;'
+    ));
+    $form->addElement('advanced_settings', Display::return_icon('synthese_view.gif') . ' ' . get_lang('ActivityCoach'));
+} else {
+    $form->addElement('text', 'coach_username', get_lang('CoachName'), array(
+        'class' => 'span4',
+        'maxlength' => 50,
+        'onkeyup' => "xajax_search_coachs(document.getElementById('coach_username').value)",
+        'id' => 'coach_username'
+    ));
+}
+
+$form->addRule('coach_username', get_lang('ThisFieldIsRequired'), 'required');
+$form->add_html('<div id="ajax_list_coachs"></div>');
+
+$form->add_select('session_category', get_lang('SessionCategory'), $categoriesOptions, array(
+    'id' => 'session_category',
+    'class' => 'chzn-select',
+    'style' => 'width:350px;'
+));
+
+$form->addElement('advanced_settings','<a class="btn-show" id="advanced_parameters" href="javascript://">'.get_lang('DefineSessionOptions').'</a>');
+
+$form->addElement('html','<div id="options" style="display:none">');
+
+$form->addElement('text', 'nb_days_acess_before', array('', '', get_lang('DaysBefore')), array(
+    'style' => 'width: 30px;',
+    'value' => $nb_days_acess_before
+));
+
+$form->addElement('text', 'nb_days_acess_after', array('', '', get_lang('DaysAfter')), array(
+    'style' => 'width: 30px;',
+    'value' => $nb_days_acess_after
+));
+
+$form->addElement('html','</div>');
+
+$form->addElement('checkbox', 'start_limit', '', get_lang('DateStartSession'), array(
+    'onchange' => 'disable_starttime(this)',
+    'id' => 'start_limit'
+));
+
+$form->addElement('html','<div id="start_date" style="display:none">');
+
+$startDateGroup = array ();
```

Comment:
You should not use this kind of tricky code to handle dates.
You can use:

<pre>$form->addElement('date_picker', 'start_date');</pre>

See example here:

https://github.com/chamilo/chamilo-lms/blob/1.9.x/main/calendar/agenda.lib.php#L1806